Makefile

```diff
@@ -16,17 +16,25 @@ clean: ## Remove generated byte code, coverage reports, and build artifacts
 	rm -fr build/
 	rm -fr dist/
 	rm -fr *.egg-info
+	@echo "The project has been cleaned successfully."
 
-selfcheck: ## check that the Makefile is well-formed
+selfcheck: ## Check that the Makefile is well-formed
 	@echo "The Makefile is well-formed."
 
-requirements:
-	pip install -q -r requirements.txt --exists-action w
+test_requirements: ## Install requirements needed by test environment
+	pip install -q -r requirements/quality.txt --exists-action w
+	pip install -q -r requirements/test.txt --exists-action w
+
+requirements: test_requirements ## Installs all requirements needed by developmenent and test environments
+	pip install -q -r requirements/base.txt --exists-action w
```

Comment:
@symbolist

> I think we can make this quality.txt so that it installs everything needed for dev.

I modified this to address your point but in the same way. I edited the requirements' files and removed the dependencies because it was confusing the order of which depends on which.

> Also below in `quality` we need to change `pep8` with `pycodestyle`.

Addressed

> And actually run the tests in test + generate diff-cover. See https://github.com/open-craft/blockstore/blob/master/Makefile#L53

Addressed

> Also need to the add the commands to `.PHONY:`

Will be adding more when we add the i18n support.